src/date-fns/_lib/format/longFormatters/index.ts

```diff
@@ -1,38 +1,41 @@
-import type { FormatLong } from '../../../locale/types';
+import type { FormatLong } from "../../../locale/types";
 
 type LongFormatter = (pattern: string, formatLong: FormatLong) => string;
 
 const dateLongFormatter: LongFormatter = (pattern, formatLong) => {
   switch (pattern) {
-    case 'P':
-      return formatLong.date({ width: 'short' });
-    case 'PP':
-      return formatLong.date({ width: 'medium' });
-    case 'PPP':
-      return formatLong.date({ width: 'long' });
-    case 'PPPP':
+    case "P":
```

Comment:
should we apply our prettier formatting?  most of the changes in this file are just formatting changes